opensoundscape/annotations.py

```diff
@@ -169,6 +169,14 @@ def from_raven_files(
         # update defaults with any user-specified mappings
         column_mapping_dict.update(column_mapping_dict or {})
 
+        standard_columns = [
```

Comment:
I realized it was defined twice in the original code too, but we should fix it. I think this is the use case for "class attribute":
```
class BoxedAnnotations:
    standard_columns = (...) # tuple because lists are mutable! 
    def __init__(...)...
    ....
        columns = list(BoxedAnnotations.standard_columns) # new copy, as a mutable list
        .... edit columns as needed...
```